src/App.jsx

```diff
@@ -83,51 +90,60 @@ function App() {
     });
   });
 
+  onCleanup(() => {
+    window.removeEventListener("resize", handleWindowResize);
```

Comment:
Why are we having this event handler in onCleanup? Coz I belive the cleanup will be called for App.jsx only when is is removed from DOM (like App closed/ redirected during Login). In that case the Empty view will not be rendered as the parent node is removed from DOM

Reply:
shouldn't we remove all the event listeners from the app even if the node is removed from DOM?